README.md

```diff
@@ -8,3 +8,37 @@ Install it from [npm](https://www.npmjs.com/package/nodejs_steambot) or check ou
 # Support
 
 Report bugs on the [issue tracker](https://github.com/jacobtruman/nodejs_steambot/issues).
+
+If you need help extracting your steam file, contact [LeeTheGayKid](http://www.steamcommunity.com/id/jingyong) <b>You will need your phone to be rooted !</b>
```

Comment:
I do not see how this is relevant. I also do not want a link to a steam profile anywhere in the documentation.